templates/splash/api-en.html

```diff
@@ -7,7 +7,7 @@
 		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html" }
 	],
 	"secondlevel": false,
-	"dateModified": "2022-09-28",
+	"dateModified": "2022-10-05",
 	"share": "false"
 }
 ---
```

Comment:
We will need to add a section about the unsupported pattern, like the version 1.